backend/terry/server.py

```diff
@@ -48,7 +48,8 @@ def __init__(self):
         self.router = Map(
             [
                 Rule("/contest", methods=["GET"], endpoint="info#get_contest"),
-                Rule("/input/<input_id>", methods=["GET"], endpoint="info#get_input"),
+                Rule("/input/<input_id>",
```

Comment:
I don't know which formatter you used, but it doesn't look like the correct one. Can you remove these changes?